library/src/nvidia_detail/hipblas.cpp

```diff
@@ -4605,6 +5441,35 @@ catch(...)
     return exception_to_hipblas_status();
 }
 
+// rotmg_64
+hipblasStatus_t hipblasSrotmg_64(
+    hipblasHandle_t handle, float* d1, float* d2, float* x1, const float* y1, float* param)
+try
+{
+    // cuBLAS documentation claims support here but doesn't seem to have it in cuda 12.3
+    // #if CUBLAS_VER_MAJOR >= 12
+    //     return hipCUBLASStatusToHIPStatus(
+    //         cublasSrotmg_64((cublasHandle_t)handle, d1, d2, x1, y1, param));
+    // #else
+    return HIPBLAS_STATUS_NOT_SUPPORTED;
+    // #endif
```

Comment:
Well you could do what we do internally and just call their non _64 function.  We just added it for a more consistent  API.

Reply:
Ah yeah true I'll do that.